packages/playground/src/utils/app-list.ts

```diff
@@ -0,0 +1,17 @@
+import { AppDefinition } from "../types";
+
+export default {
+  // TODO: How do we get a list of available apps?
+  "0x822c045f6F5e7E8090eA820E24A5f327C4E62c96": {
```

Comment:
How is the hash computed?

Reply:
@snario This is just mocked data, but it's the address where the dapp's smart contract is deployed.